### ISSUE TITLE
A generated fish script doesn't complete an option after an argument.

### ISSUE THREAD

**Issue by mtj0928** [apple/swift-argument-parser#534]
A fish script that is generated by `--generate-completion-script` doesn't complete an option after an argument.
For example, `repeat` command which is contained in the repository as an example can run the following command.
```shell
$ repeat foo --count 10 
```
However, `--count` is not completed even though `$ repeat foo --c` is input and a tab key is pressed.

**ArgumentParser version:** `main`
**Swift version:**  
```
swift-driver version: 1.62.15 Apple Swift version 5.7.1 (swiftlang-5.7.1.135.3 clang-1400.0.29.51)
Target: arm64-apple-macosx13.0
```

### Checklist
- [x] If possible, I've reproduced the issue using the `main` branch of this package
- [x] I've searched for [existing GitHub issues](https://github.com/apple/swift-argument-parser/issues)

### Steps to Reproduce
1. Build this repository
```shell 
$ swift build -c release
```
2. Add the release directory into your $PATH.
```shell
$ set -x PATH (string join "" (pwd) "/.build/release/") $PATH  
```

3. Generate a fish completion script.
```
$ repeat --generate-completion-script fish >> ~/.config/fish/completions/repeat.fish
```

4. Type `repeat foo --` and press a tab key to try to complete options.

### Expected behavior
Options that `repeat` supports are shown.

### Actual behavior
Nothing is shown.

**Comment by hassila** [comment]
See https://github.com/apple/swift-argument-parser/issues/376 also

**Comment by natecook1000** [comment]
Closed by #535.
